### PR TITLE
Adds The Sturdy Shako (HoS Hat)

### PR DESCRIPTION
**Pull request by Thunder12345**
## About The Pull Request

Adds a new craftable HoS shako, made from the standard peaked cap and some plasteel and gold.

shako

Moves the HoS cap into a subtype `/obj/item/clothing/head/hats/hos/cap` to avoid catching the beret in the crafting recipe, the base type `/obj/item/clothing/head/hats/hos` is now a generic subtype which should never exist.

(Yes there's a TF2 reference included with the hat)

## Why It's Good For The Game

Adds an additional level of style for an HoS willing to put a little effort into gathering the required materials. As it is crafted as a one-to-one trade with another headgear there is no additional spare headgear introduced. 

## Changelog
:cl:
add: The Nanotrasen Costuming Department has released specifications for how to convert the Head of Security's cap into a shako.
/:cl:

### CODE REVIEW

**Comment by san7890** [comment]
2 ref

**Comment by Likteer** [comment]
Tough chance, but could I ask for a novelty version?

**Comment by Thunder12345** [comment]
> Tough chance, but could I ask for a novelty version?

No, make a real one

File: code/datums/components/crafting/tailoring.dm [hunk 1]
```diff
@@ -304,3 +304,16 @@
...
 	category = CAT_CLOTHING
+
+/datum/crafting_recipe/sturdy_shako
+	name = "Sturdy Shako"
+	result = /obj/item/clothing/head/hats/hos/shako
+	tool_behaviors = list(TOOL_WELDER, TOOL_KNIFE)
+	time = 5 SECONDS
+	reqs = list(
+		/obj/item/clothing/head/hats/hos/cap = 1,
+		/obj/item/stack/sheet/plasteel = 2, //Stout shako for two refined
```
Comment (Time-Green) [inline code/datums/components/crafting/tailoring.dm:315]: thank you

File: code/modules/clothing/head/jobs.dm [hunk 2]
```diff
@@ -169,11 +169,15 @@
 
 //Security
 /obj/item/clothing/head/hats/hos
+	name = "generic head of security hat"
+	desc = "Please contact the NanoTrasen Costuming Department if found."
+	armor_type = /datum/armor/hats_hos
+	strip_delay = 80
```
Comment (Jacquerel) [inline code/modules/clothing/head/jobs.dm:175]:
```suggestion
	strip_delay = 8 SECONDS
```
Just while you happen to be here

**Review by Jacquerel — CHANGES_REQUESTED** [review]

File: code/modules/clothing/head/jobs.dm [hunk 3]
```diff
@@ -169,11 +169,15 @@
 
 //Security
 /obj/item/clothing/head/hats/hos
+	name = "generic head of security hat"
+	desc = "Please contact the NanoTrasen Costuming Department if found."
```
Comment (Fikou) [inline code/modules/clothing/head/jobs.dm:173]:
```suggestion
	desc = "Please contact the Nanotrasen Costuming Department if found."
```

**Review by Fikou — CHANGES_REQUESTED** [review]

**Pull request merged**